packages/item/src/styled/index.ts

```diff
@@ -13,7 +13,7 @@ import { IPrefixPostfixProps, IContentProps, IRootProps } from '../interfaces';
 export const Root = styled.div<IRootProps>`
   ${getBaseStyle}
   box-sizing: border-box;
-  cursor: pointer;
+  cursor: ${({ theme, appearance, baseAppearance }): string => itemAppearanceTheme(theme, appearance, baseAppearance, 'cursor')};
```

Comment:
If you remove this line, everything will work the same

Reply:
but why we should use it with "border-radius" (18th line)? because in the theme we use only numbers for borders and here we have to add "px"?

Reply:
> because in the theme we use only numbers for borders and here we have to add "px"
yep. I think in the future we will remove this line